content/blog/2022/05/psp-migration-docs.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Have you already migrate your Kubernetes PodSecurityPolicy?"
+authors:
+- José Guilherme Vanz
+date: 2022-05-16
+---
+
+If you use the deprecated Kubernetes PodSecurityPolicy (a.k.a PSP), you should
+wonder what to do after the Kubernetes v1.25 version. When the PSP will be removed.
+If this in mind, the Kuberwarden team wrote a [documentation](https://docs.kubewarden.io/psp-migration.html)
+to help users migrate away from PSPs to Kuberwarden policies.
+
```

Comment:
Can we briefly describe what we've mentioned in the docs for a quick overview? The ending seems to be too abrupt

Reply:
Turning on my random text generator:

> As you know, the original Pod Security Policies had many configuration knobs. The Kubewarden team created a series of policies that offer a 100% feature parity with all the soon to be dropped Pod Security Policies.
> This section of our documentation highlights all these policies and guides you through their recommended settings.
>
> On this topic, do you know our helm chart provides a flag that enforces a chosen selection of Pod Security Policies? This makes even easier to keep your clusters safe!

This is just an idea, feel free to change it as you wish

Reply:
I like that paragraph from Flavio.

Reply:
I've just added it in the text. ;)